addons/bus/static/src/js/services/async_service.js

```diff
@@ -0,0 +1,41 @@
+odoo.define('bus.asyncService', function (require) {
+"use strict";
+
+const AbstractService = require('web.AbstractService');
+const { serviceRegistry, bus, _lt, _t } = require('web.core');
+
+
+const notifHead = _lt("Background task created");
+const notifCreatedBody = _lt("The task %s has been scheduled for processing and will start shortly.");
+
+const asyncService = AbstractService.extend({
```

Comment:
- IMO `AsyncService` is too broad naming. Suggestion: `AsyncJobService`.
- classes start with capital letter, so `AsyncJobService`